src/Compilers/Core/CodeAnalysisTest/ObjectSerializationTests.cs

```diff
@@ -548,7 +548,7 @@ private static void TestWritingPrimitiveArrays(ObjectWriter writer)
             var inputString = new string[] { "h", "e", "l", "l", "o" };
 
             writer.WriteValue(inputBool);
-            writer.WriteValue(inputByte);
+            writer.WriteValue((object)inputByte);
```

Comment:
not certain i understand this bit of the change.

Reply:
There is no best conversion from `byte` to `object`/`Span<byte>`